zkevm-circuits/src/evm_circuit/execution/memory.rs

```diff
@@ -214,9 +214,7 @@ mod test {
                 + OpcodeId::PUSH32.as_u64()
                 + OpcodeId::PUSH32.as_u64()
                 + gas_cost,
-            // we have to disable state circit now, since the memory size used
-            // here is too large
-            enable_state_circuit_test: false,
+            enable_state_circuit_test: true,
```

Comment:
We could even remove this field entirely if we're not considering to test without this anymore.